lib/ecl/build/install-ecl.sh

```diff
@@ -1,16 +1,8 @@
 #!/usr/bin/env sh
 set -e
 
-wget -qO- https://gmplib.org/download/gmp/gmp-6.1.2.tar.xz | tar -xJf - --one-top-level=gmp --strip-components=1
```

Comment:
Also, `--one-top-level=` is bash-only, so I changed it to a portable way. Everything else in this PR is pure copy-paste.